code/clue.js

```diff
@@ -7,53 +7,220 @@ const mrGreen = {
+const theater = 'Theater'
+const guestHouse = 'Guest House'
+const patio = 'Patio'
 
-// NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
+console.log(mrGreen.description)
+console.log(rope.weight)
 
+// NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
 const suspects = [
   mrGreen,
-  mrsWhite
-  // ...  and the rest
-]
+  professorPlum,
+  missScarlet,
+  mrsPeacock,
+  colonelMustard,
+  mrsWhite,
+];
+
+const weapons = [
+  rope,
+  knife,
+  candleStick,
+  dumbBell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol,
+];
 
-const weapons = []
+const rooms = [
+  diningRoom,
+  conservatory,
+  kitchen,
+  study,
+  library,
+  billiardRoom,
+  lounge,
+  ballroom,
+  hall,
+  spa,
+  livingRoom,
+  observatory,
+  theater,
+  guestHouse,
+  patio
+];
 
-const rooms = []
+console.log(suspects);
+console.log(weapons);
+console.log(rooms);
+console.log()
 
 // THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FROM THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
 // YOU DON'T NEED TO CHANGE THIS, JUST TRY TO UNDERSTAND IT. AND HOW TO USE IT.
 const randomSelector = array => {
   return array[Math.floor(Math.random() * array.length)]
 }
-
 // CREATE AN OBJECT THAT KEEPS THE MYSTERY.
 // With a killer, a weapon and a room.
 // The values will be set later.
-
+let mystery = {
+  killer: '',
+  room: '',
+  weapon: ''
+}
+console.log(mystery)
 // This function will be invoked when you click on the killer card.
 const pickKiller = () => {
   // This will randomly select a killer from the suspects. And add that to the mystery object.
   mystery.killer = randomSelector(suspects)
-
   // This will change the background color of the card to the one connected to the chosen killer and show the full name of the killer. Feel free to add more things to show about the killer.
   document.getElementById('killerCard').style.background = mystery.killer.color
-  document.getElementById(
-    'killerName'
-  ).innerHTML = `${mystery.killer.firstName} ${mystery.killer.lastName}`
+  document.getElementById('killerName').innerHTML = `${mystery.killer.firstName} ${mystery.killer.lastName}`
+  document.getElementById('killerImage').src = mystery.killer.image
 }
+document.getElementById('killerCard').onclick = pickKiller //Pick Killer
 
 // CREATE FUNCTIONS pickWeapon and pickRoom in a similar way.
 
+const pickWeapon = () => {
+  mystery.weapon = randomSelector(weapons)
+  document.getElementById('weaponName').innerHTML = `${mystery.weapon.name}`
+}
+document.getElementById('weaponCard').onclick = pickWeapon //Pick Weapon
+const pickRoom = () => {
+  mystery.room = randomSelector(rooms)
+  document.getElementById('roomName').innerHTML = `${mystery.room}`
+}
+document.getElementById('roomCard').onclick = pickRoom // Pick Room
+
 // STEP 4 - CREATE A FUNCTION revealMystery that will be invoked when you click that button. It should show something like:
 // 'The murder was committed by Jacob Green, in the living room with a rope.'
```

Comment:
I'd like to encourage you to either update or remove these commented out lines. Update to reflect what's happening with the code so that someone viewing it can make sense of it or remove to reduce clutter.